app/src/androidTest/java/com/github/orkest/SpotifyMultimediaTest.kt

```diff
@@ -0,0 +1,61 @@
+package com.github.orkest
+
+import android.util.Log
+import androidx.test.core.app.launchActivity
+import com.github.orkest.data.SpotifyMultimedia
+import com.github.orkest.ui.sharing.SharingComposeActivity
+import org.junit.Before
+import org.junit.Test
+
+class SpotifyMultimediaTest {
+
+    private var accessToken : String = String()        // access token for Spotify API
+    private var authorizationCode : String = String()  // authorization code for token request
+
+    private val scenario = launchActivity<SharingComposeActivity>()
+
+    // in case spotify authorization fails on CI, bypass test.
+    private var authBypass = false
+
+    @Before
+    fun setup(){
+        // sleep for 5 seconds
+        Thread.sleep(5000)
+        scenario.onActivity {
+            this.accessToken = it.accessToken
+            Log.d("SPOTIFY TOKEN", this.accessToken)
+        }
```

Comment:
A good alternative for Thread.sleep() is to use CountDownLatch to wait for the activity to be created before proceeding with the tests